src/bp/sdk/botpress.d.ts

```diff
@@ -648,6 +648,10 @@ declare module 'botpress/sdk' {
       readonly credentials?: any
       /** When false, some properties used by the debugger are stripped from the event before storing */
       debugger?: boolean
+      /** Track processing steps during the lifetime of the event  */
+      processing?: {
```

Comment:
Thinking about it I would not expose `addStep` in the event.  We want to make the event a data structure instead of a class.  Only the core uses addStep so let's put this outside?